.github/workflows/deploy.yml

```
@@ -0,0 +1,84 @@
+      run: npm ci
+
+    - name: Build React app
+      run: npm run build
+
+    # 4. Docker 설정
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    # 5. Azure 로그인
+    - name: Azure Login
+      uses: azure/login@v1
```

actionlint (1.7.7)

the runner of "azure/login@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)